lib/routes/cashboxRoutes.js

```diff
@@ -10,13 +10,12 @@ function notifyCashboxRemoval (req, res, next) {
   return getMachine(req.deviceId)
     .then(machine => {
       loadLatestConfig()
-        .then(async config => {
+        .then(config => {
           const cashInSettings = getCashInSettings(config)
-          console.log(cashInSettings)
           if (cashInSettings.cashboxReset === 'Automatic') {
-            await cashbox.createCashboxBatch(req.deviceId, machine.cashbox)
-            await setMachine({ deviceId: req.deviceId, action: 'emptyCashInBills' })
-            return res.status(200).send({ status: 'OK' })
+            return cashbox.createCashboxBatch(req.deviceId, machine.cashbox)
+              .then(() => setMachine({ deviceId: req.deviceId, action: 'emptyCashInBills' }))
+              .then(() => res.status(200).send({ status: 'OK' }))
```

Comment:
This status code 200 is only being sent if the cashboxReset is set to automatic, which shouldn't happen, as it should always send 200 independently of the option.
Move line 18 outside of the if statement